x-pack/legacy/plugins/uptime/public/components/functional/kuery_bar/index.tsx

```diff
@@ -27,6 +28,9 @@ interface State {
   isLoadingIndexPattern: boolean;
 }
 
+const getAutocompleteProvider = (language: string) =>
```

Comment:
`getAutocompleteProvider` is just a call to `data.autocomplete.getProvider`.
I think we should replace the usage of `getAutocompleteProvider` with the call in all relevant places.